types/split-file/tsconfig.json

```
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
```

```suggestion
            "es6"
```
you've @types/ndoe as dependency, why `dom` lib has been added?

---

Because `console.log` would give errors otherwise

---

I will try commiting the suggestion